samples/multiplatform/kmp-android-app/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.multiplatform")
```

Comment:
Do we actually need the multiplatform plugin here ? This module will always be just Android, right ?

Reply:
Addressed here: https://github.com/apollographql/apollo-android/pull/2136/commits/3cb048ceaa38c5164bf8cd946160d6f4a7457b60